.idea/libraries/Gradle__androidx_activity_activity_1_1_0_aar.xml

```diff
@@ -1,14 +1,14 @@
 <component name="libraryTable">
```

Comment:
You shouldn't be committing these files. Look into what files should be included in `gitignore` and remove them if they are already version controlled.

Reply:
yes. Im deleting them.